### DISCUSSION TITLE
Allow changing selected tabs

### DISCUSSION THREAD

**Discussion by erikwestlund** [livewire/flux#643] [Feature Requests] [answered]
I have multi-tab forms. I want a way to select a tab programmatically, particularly when form validation fails. I'd like to be able to tell it which tab to select.

It would be nice to have an API for this. 

I realize that may feel out of scope, at least for now in early development.  So given that, how would others handle this?

I'm thinking perhaps the easiest way is just to select the tab button and click it using javascript. That would trigger everything related to it.  But I'm wondering if anyone has a less hacky way.

**Comment by jeffchown** [comment, marked as answer]
@erikwestlund If I understand your post correctly, you can accomplish this using a property that you `wire:model` to your tabs, e.g.:

https://github.com/livewire/flux/discussions/580#discussioncomment-11126885

Docs: https://fluxui.dev/docs/patterns#data-binding

**Comment by erikwestlund** [comment]
Yes, this is perfect and exactly what I needed. Sorry, missed it -- might be worth documenting on the Tabs component page.
